docs/3-react/3-fine-grained-reactivity.mdx

```diff
@@ -226,14 +226,18 @@ Props:
 </Show>
 ```
 
+<br />
+
 ```jsx
 import { Show } from '@legendapp/state/react'
 import { computed } from '@legendapp/state'
```

Comment:
we should maybe remove this import if we won't use computed.

I personally use the computed higher-order function when running my show components.